pkg/resolution/resolver/hub/resolver.go

```diff
@@ -174,7 +298,16 @@ func (*ResolvedHubResource) Annotations() map[string]string {
 // Source is the source reference of the remote data that records where the remote
 // file came from including the url, digest and the entrypoint.
 func (rr *ResolvedHubResource) Source() *pipelinev1beta1.ConfigSource {
-	return nil
+	h := sha256.New()
+	h.Write(rr.Content)
+	sha256CheckSum := hex.EncodeToString(h.Sum(nil))
+
+	return &pipelinev1beta1.ConfigSource{
+		URI: rr.URL,
+		Digest: map[string]string{
+			"sha256": sha256CheckSum,
+		},
+	}
```

Comment:
> Just curious why we don't use https://pkg.go.dev/crypto/sha256#Sum256 here? Any difference between Sum256 and the current approach we used?

Thanks for the comments, @chuangw6! I think the difference is that it used hex encoding of the check sum. It could be easier to interpret if we want to display the value  based on the post here: https://stackoverflow.com/a/10701951.

Happy to remove the encoding part if not necessary to you  :)

Reply:
@chuangw6 I wouldn't worry about it too much - these are functional equivalent.

Reply:
SG! Keeping it as is makes sense to me! Thanks @QuanZhang-William @wlynch !

Reply:
Cool! @chuangw6 , added the rr.URL value, PTAL.